src/bin/registerRule.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('commander');
+const Web3 = require('web3');
+const OpenST = require('@openstfoundation/openst.js');
+
+const ChainConfig = require('../config/chain_config');
+const logger = require('../logger');
+
+const { version } = require('../../package.json');
+
+program
+  .version(version)
+  .name('RegisterRule')
+  .arguments('<config> <ruleName> <ruleAddress> <ruleAbi>')
+  .description('An executable to register rules in tokenrules.')
+  .action(
+    async (config, ruleName, ruleAddress, ruleAbi) => {
+      const chainConfig = new ChainConfig(config);
+      const registerRuleTxOptions = {
```

Comment:
You should use `connected::run()` to assert correct signing of transactions and closing of the connection.